web/src/components/ui/icons.tsx

```diff
@@ -1139,4 +1139,21 @@ export const FontIcon = ({ type, ...props }: FontIconProps) => (
   <span aria-hidden="true" data-icon={FONT_ICONS[type]} {...props} />
 );
 
+export const XIcon = () => {
```

Comment:
There's an icon in this file with the same name (actually it has a typo it is spelt as _XICon_ instead of _XIcon_). 

Can you change the name of the other icon (with the typo) to `XMarkIcon`? You'll also have to update the name of the icon to `XMarkIcon` in the `example-content-voice.tsx` file because that's where it is used.

Reply:
ok will do that..